helm/alfresco-content-services/values.yaml

```diff
@@ -538,6 +551,91 @@ elasticsearch:
   enabled: true
   replicas: 1
   clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+elasticsearchAudit:
+  # -- Enables the embedded elasticsearch cluster for alfresco-audit-storage
+  enabled: true
+  nameOverride: elasticsearch-aas
+  replicas: 1
+  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+  clusterName: elasticsearch-aas
+  extraEnvs:
+    - name: ELASTIC_USERNAME
+      valueFrom:
+        secretKeyRef: 
+          name: *aas_elasticsearch_secretName
+          key: AUDIT_ELASTICSEARCH_USERNAME
+    - name: ELASTIC_PASSWORD
+      valueFrom:
+        secretKeyRef: 
+          name: *aas_elasticsearch_secretName
+          key: AUDIT_ELASTICSEARCH_PASSWORD
+  ingress:
```

Comment:
The ingress is disabled by default, but that's a good thing to have it explicitly disabled here. Though further config is not needed here. We should maybe just link to the upstream chart doc for ingress config details: